system/formats/copytoram-iso.nix

```diff
@@ -0,0 +1,20 @@
+# derived from https://github.com/nix-community/nixos-generators/blob/master/formats/install-iso.nix
+{ config, lib, modulesPath, ... }:
+{
+  imports = [
+    "${toString modulesPath}/installer/cd-dvd/installation-cd-base.nix"
+  ];
+
+  # for installer
+  isoImage.isoName = "nixos.iso";
+
+  # override installation-cd-base and enable wpa and sshd start at boot
+  systemd.services.wpa_supplicant.wantedBy = lib.mkForce [ "multi-user.target" ];
+  systemd.services.sshd.wantedBy = lib.mkForce [ "multi-user.target" ];
```

Comment:
I wonder are these required since we already enable openssh? wpa_supplicant should probably not needed either, since none of the machines right now have WiFi.